.travis.yml

```diff
@@ -18,6 +18,26 @@ script:
 after_success:
   - mvn jacoco:report jacoco:report-integration coveralls:report
 
+before_deploy:
+  - git config --global user.email "builds@travis-ci.com"
+  - git config --global user.name "Travis CI"
+  - export GIT_TAG=$TRAVIS_BRANCH-v1.0.$TRAVIS_BUILD_NUMBER
+  - git tag $GIT_TAG -a -m "Generated tag from TravisCI for build $TRAVIS_BUILD_NUMBER"
+  - git push -q https://$API_KEY@github.com/rackerlabs/blueflood --tags
+
+deploy:
+  skip_cleanup: true
+  provider: releases
+  api_key: 
+    secure: Eue8cRnapBQim2wArFkR88r+SJHe4Qs5gcor8QCWJuFltJcRG7JCu+y7FAjeYSCBqXGNVENirRjV5YnfdQS4uNKhb3m3WalneJDGCUtk82VdeJiNwV52IjKJEmpTUQu+bun1aoU0+ZY2PkzFW8S1lpCzj+n/YH3LWgIiXayiOhOzL2Yq1VJp2/BVnEZvKRpVlN55SwFPTkWVta2qKpr0b7KfNpPLQArQqJUExrHMQxFb2iM1Yb0xp2C2HQIYSr3xYw9eR5TjFI2NEZwfzhsrJ5M/ZVbkM5UAOranHGiQvJ2Eha8o0XHqvjOCPcJUVXJ7M9+xRA5THMVBrLCMZOH9LIskwHQxx9318b1jfw2uM+fwEZwqfyakCN/pmaQFxzzBmVDsFwlkoqzYIRimanOlLhwyNY9Nkc0XN40ksvIdmIoG0llOtZFQf168WsWUT5GR1QQrNe1sAgWdPQ51suxEcL0HITwlna4VUakdbNxBZjZ5+//pTMohH5aWlkJSt+4oJIsq0tUpDonGQ7pJ9iR12cISiXmjBAEFDKtPM2JJsBuP73kLmgHS11lUY0sFtz5oyR9NjBr2Gjrs15XCFqZwn21SiTzunHo9e3k2oiU74g3B7UZ+YiiKRhnv5gTu7Gps4MgpVlIsOPU4h9ygZQJga2odoJKKMMFQlyCsGHyo54U=
```

Comment:
What's the best way to configure this key? I don't like it being there in the yaml file. Can it be there in blueflood.conf?